unittests/list_restriction/test_list_restriction.cpp

```diff
@@ -0,0 +1,273 @@
+#include <wsdl.h>
+
+#include <messagehandler.h>
+#include <parsercontext.h>
+
+#include <converter.h>
+#include <creator.h>
+#include <settings.h>
+
+#include <compiler.h>
+#include "httpserver_p.h"
+
+#include <QtTest>
```

Comment:
This includes all of QtTest *and* all of QtCore.

I'll change it to <QTest> instead (and add whatever is missing)

Reply:
(you forgot to remove the 't' so this is still a full-module-include)